xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -54,6 +54,173 @@
+    {0xA930, 0xA95F, "NotoSansRejang-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Rejang"},
+    {0xA960, 0xA97F, "NotoSansKR-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Hangul Jamo Extended-A"},
+    {0xA980, 0xA9DF, "NotoSansJavanese-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Javanese"},
+    {0xA9E0, 0xA9FF, "NotoSansMyanmar-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Myanmar Extended-B"},
+    {0xAA00, 0xAA5F, "NotoSansCham-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Cham"},
+    {0xAA60, 0xAA7F, "NotoSansMyanmar-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Myanmar Extended-A"},
+    {0xAA80, 0xAADF, "NotoSansTaiViet-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Tai Viet"},
+    {0xAAE0, 0xAAFF, "NotoSansMeeteiMayek-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Meetei Mayek Extension"},
+    {0xAB00, 0xAB2F, "NotoSansEthiopic-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Ethiopic Extension-A"},
+    {0xAB30, 0xAB6F, "NotoSans-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Latin Extended-E"},
+    {0xAB70, 0xABBF, "NotoSansCherokee-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Cherokee Supplement"},
+    {0xABC0, 0xABFF, "NotoSansMeeteiMayek-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Meetei Mayek"},
+    {0xAC00, 0xD7AF, "NotoSansKR-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Hangul Syllable"},
+    {0xD7B0, 0xD7FF, "NotoSansKR-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Hangul Jamo Extended-B"},
+    //{0xD800, 0xDB7F, "", false, CFontTable::ALIGN::LTOR, "High Surrogates"},
+    //{0xDB80, 0xDBFF, "", false, CFontTable::ALIGN::LTOR, "Private Use High Surrogates"},
+    //{0xDC00, 0xDFFF, "", false, CFontTable::ALIGN::LTOR, "Low Surrogates"},
+    //{0xE000, 0xF8FF, "", false, CFontTable::ALIGN::LTOR, "Private Use Area"},
+    {0xF900, 0xFAFF, "NotoSansJP-Regular.ttf", true, CFontTable::ALIGN::LTOR, "CJK Compatibility Ideographics"},
+    {0xFB00, 0xFB4F, "NotoSans-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Alphabetic Presentation Forms"},
+    {0xFB50, 0xFDFF, "NotoSansArabic-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Arabic Presentation Forms"},
+    //{0xFE00, 0xFE0F, "", false, CFontTable::ALIGN::LTOR, "Variation Selectors"},
+    {0xFE10, 0xFE1F, "NotoSansSC-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Vertical Forms"},
+    {0xFE20, 0xFE2F, "NotoSans-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Combining Half Masks"},
+    {0xFE30, 0xFE4F, "NotoSansSC-Regular.ttf", true, CFontTable::ALIGN::LTOR, "CJK Compatibility Forms"},
+    {0xFE50, 0xFE6F, "NotoSansSC-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Small Form Variants"},
+    {0xFE70, 0xFEFF, "NotoSansArabic-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Arabic Presentation Forms-B"},
+    {0xFF00, 0xFFEF, "NotoSansSC-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Halfwidth and Fullwidth Forms"},
+    {0xFFF0, 0xFEFF, "NotoSans-Regular.ttf", true, CFontTable::ALIGN::LTOR, "Specials"},
```

Comment:
Have now added the mostly complete table with set of unicodes and reference to related font.

Would look also to implement unicodes with more as 16 bit and to allow emojis with it.

The selection as worked good during test, only need to look to become better alignment, some fonts are a bit lower or upper as the normal Latin words. Also was in Arabic the alignment not to right.